package/Dockerfile

```diff
@@ -117,7 +117,6 @@ ENV AUDIT_LOG_MAXAGE 10
 ENV AUDIT_LOG_MAXBACKUP 10
 ENV AUDIT_LOG_MAXSIZE 100
 ENV AUDIT_LEVEL 0
-VOLUME /var/log/auditlog
```

Comment:
I'm assuming we can remove this volume because it's covered below, not sure if this has repercussions for upgrade scenario.